PyInstaller/compat.py

```diff
@@ -319,6 +319,9 @@ def exec_command(*cmdargs, **kwargs):
         print(e, file=sys.stderr)
         print('--' * 20, file=sys.stderr)
         raise ExecCommandFailed("Error: Executing command failed!") from e
+    except subprocess.TimeoutExpired:
+        proc.kill()
```

Comment:
note: there is a race here, but since we're re-raising the exception, the worst that can happen is another exception